File: Solutions/20256/c++/program.cpp
// Copy from Quera

#include <iostream>
using namespace std;

int main()
{
    char c1, c2, c3, c4, c5;
    cin >> c1 >> c2 >> c3 >> c4 >> c5;
    int R = 0, Y = 0, G = 0;
    // This piece of code repeats 5 times!
    // 1
    if (c1 == 'R')
        R = R + 1;
    if (c1 == 'Y')
        Y = Y + 1;
    if (c1 == 'G')
        G = G + 1;

    // 2
    if (c2 == 'R')
        R = R + 1;
    if (c2 == 'Y')
        Y = Y + 1;
    if (c2 == 'G')
        G = G + 1;

    // 3
    if (c3 == 'R')
        R = R + 1;
    if (c3 == 'Y')
        Y = Y + 1;
    if (c3 == 'G')
        G = G + 1;

    // 4
    if (c4 == 'R')
        R = R + 1;
    if (c4 == 'Y')
        Y = Y + 1;
    if (c4 == 'G')
        G = G + 1;

    // 5
    if (c5 == 'R')
        R = R + 1;
    if (c5 == 'Y')
        Y = Y + 1;
    if (c5 == 'G')
        G = G + 1;

    if (R >= 3 || (Y >= 2 && R >= 2) || G == 0)
        cout << "nakhor lite\n";
    else
        cout << "rahat baash\n";
    return 0;
}

File: Solutions/20256/java/program.java
// Copy from Quera

import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        String label = s.next();
        int R = 0, G = 0, Y = 0;
        
        if(label.charAt(0) == 'R') R++;
        else if(label.charAt(0) == 'G') G++;
        else Y++;
        
        if(label.charAt(1) == 'R') R++;
        else if(label.charAt(1) == 'G') G++;
        else Y++;
        
        if(label.charAt(2) == 'R') R++;
        else if(label.charAt(2) == 'G') G++;
        else Y++;
        
        if(label.charAt(3) == 'R') R++;
        else if(label.charAt(3) == 'G') G++;
        else Y++;
        
        if(label.charAt(4) == 'R') R++;
        else if(label.charAt(4) == 'G') G++;
        else Y++;
        
        if( (R>=3) || (R==2 && Y>=2) || (G==0) ) System.out.println("nakhor lite");
        else System.out.println("rahat baash");
    }
}